pkgs/development/libraries/nuspell/default.nix

```
@@ -7,6 +7,7 @@
   pkg-config,
   icu,
   catch2_3,
+  enableManpages ? !stdenv.buildPlatform.isRiscV64,
```

```suggestion
  enableManpages ? !stdenv.buildPlatform.isRiscV64 && !stdenv.buildPlatform.isLoongArch64,
```

A similar fix to this PR is already applied downstream on LoongArch64, see https://github.com/loongson-community/nixpkgs/commit/5bfec5c6a4dd4b1bed389e70f236afc90da296e3. It would be appreciated if loongarch64-linux could be added here.